R/vars_manager.R

```diff
@@ -263,6 +273,7 @@ get_matrix_var <- function(
 #' @importFrom dplyr arrange %>%
 #' @importFrom rlang parse_exprs
 #' @importFrom tibble rownames_to_column column_to_rownames
+#' @importFrom frictionless create_schema
```

Comment:
does this dependency need to be removed? looks like you just copied the source code instead

Reply:
whoops, you are correct